collections/without_all.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+
+/**
+ * Returns an array excluding all given values.
+ *
+ * Example:
+ *
+ * ```ts
+ * import { withoutAll } from "./withoutAll.ts";
```

Comment:
Looks like a typo in module path

```suggestion
 * import { withoutAll } from "./without_all.ts";
```